code/datums/skills/gaming.dm

```diff
@@ -5,6 +5,7 @@
 	modifiers = list(SKILL_PROBS_MODIFIER = list(0, 5, 10, 15, 15, 20, 25),
 				SKILL_RANDS_MODIFIER = list(0, 1, 2, 3, 4, 5, 7))
 	skillcapepath = /obj/item/clothing/neck/cloak/skill_reward/gaming
+	level_names = list("None", "Gaming Noob", "Casual Gamer", "Intermediate Gamer", "Advanced Gamer", "Pro Gamer", "Gamer Legend") //List of skill level names. Note that indexes can be accessed like so: level_names[S_NOVICE]
```

Comment:
when I made the system Ninja and me intentionally wanted a df inspired system that doesn't have meme titles for every skill because it makes it more confusing.